### DISCUSSION TITLE
More explicit presets

### DISCUSSION THREAD

**Discussion by viters** [directus/directus#12036] [Feature Requests]
Presets right now have a lot of _implicitness_, they save automatically on changes (even without any changes sometimes) and it is not clear what preset is currently used and that there are others available.

As a developer I introduce new fields in collections and I want to adjust table or other layouts accordingly. So I maintain the default presets. Although effect may not appear in administrator's or content editor's panels, because they do not use the default preset - sometimes even unintentionally.

It's concerning, because presets affects relations selecting views as well.

Moreover, it is not clear for them that presets even exist and verify what preset is active, it requires a trip to Presets settings which can be overwhelming.

**Comment by benhaynes** [comment]
Are you referring to User Bookmarks, or on the Settings > Presets page?

**Reply by viters** [reply]
Settings > Presets

**Reply by benhaynes** [reply]
Ahh, ok... this entire page is about to be overhauled, so this should be resolved then. If you have any specific feedback for that sprint, we welcome any ideas!

**Reply by viters** [reply]
Gotcha. I do not know what are your plans for that - but I have two concerns:

1. Presets are sometimes created for user even if he did not change anything on a view. It's counterintuitive and leads to him not seeing changes made to that view in default preset in the future (without purging all custom user presets in each deployment, which is also not a good idea). I think the presets should not be saved automatically. As a user, if I make changes to table view, it is not obvious, that I want to persist it. Sometimes I just play around with layout settings to get the data I need, and would like it to go back to default as soon as I finish.
2. It's not clear what preset is currently being used from collections views (in opposition to bookmarks, they are fine). Maybe the select for presets could be available somewhere nearby (maybe in Layout Options, although that is a bit hidden). Quick and exaggerated  example:
![image](https://user-images.githubusercontent.com/13100466/157478917-eda7792f-0099-4083-8ee9-7836b98518e0.png)

**Reply by benhaynes** [reply]
Thank you! I've added this feedback to our internal tracker for the Preset update. ❤️

**Reply by hamwithglasses** [reply]
Completely agree with this. Looking forward to the update here, but even simply adding the reset/revert to the default "Global Users" layout, much like you can reset a saved/named Bookmark's view, would immediately make this more intuitive.

I'd almost consider the lack of having that revert icon on the default view a bug.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
